tests/utils/test_settings.py

```diff
@@ -0,0 +1,86 @@
+from scrapy.settings import Settings
```

Comment:
There is no test to verify if the DeprecationWarning is raised and logged.

Reply:
I don't if we should test it.
Do we test log messages?
Of course this is a bit different, but I don't know if it is a valid one.

Reply:
The log is there for some reason, so we need to ensure that it is being logged as expected.